-- Fix pyproject.toml for compatibility with the setuptools version available in ports:
-- - Convert license string to PEP 621 table format.
-- - Remove license-files property (not supported by setuptools<69).
-- - Lower setuptools build requirement from >=69 to >=63.1.0.
--- pyproject.toml.orig	2026-09-09 02:17:05 UTC
+++ pyproject.toml
@@ -3,8 +3,7 @@ readme = {file = "README.md", content-type = "text/mar
 dynamic = ["version", "dependencies", "optional-dependencies"]
 description = "ModelScope: bring the notion of Model-as-a-Service to life."
 readme = {file = "README.md", content-type = "text/markdown"}
-license = "Apache-2.0"
-license-files = ["LICENSE"]
+license = {text = "Apache-2.0"}
 authors = [
   {name = "ModelScope team"},
   {email = "contact@modelscope.cn"}
@@ -43,7 +42,7 @@ modelcard = "modelscope.cli.modelcard:ModelCardCMD"
 modelcard = "modelscope.cli.modelcard:ModelCardCMD"
 
 [build-system]
-requires = ["setuptools>=69", "wheel"]
+requires = ["setuptools>=63.1.0", "wheel"]
 build-backend = "setuptools.build_meta"
 
 [tool.setuptools]
